backends/platform/android/android.mk

```diff
@@ -3,7 +3,7 @@
 # These must be incremented for each market upload
 ANDROID_VERSIONCODE = 1000
 
-ANDROID_TARGET_VERSION = 15
+ANDROID_TARGET_VERSION = 23
```

Comment:
We'll need to bump this in the future to actually get the thing published, as google [requires version 26](https://developer.android.com/distribute/best-practices/develop/target-sdk) for updates, and 29 from Nov 2019 on. But let's get all of this merged first :)